com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConverterContainer.cs

```diff
@@ -0,0 +1,18 @@
+namespace UnityEditor.Rendering
+{
+    /// <summary>
+    /// A class for different conversions.
```

Comment:
The documentation on this class and its properties is not really helpful, as it doesn't tell you anything that the code names don't already tell you. What can you do with the class? Which kinds of conversions? As for the properties, what is the effect of the values? Where are they used? etc.

Also, given how the converter API is not public yet, should this be?

Reply:
Done, Made it internal and updated doc strings